tools/common/lib.go

```diff
@@ -546,20 +546,30 @@ func ReadDlq(c *cli.Context, mClient mcli.Client) {
 
 // ReadCgBacklog reads the CG back log
 func ReadCgBacklog(c *cli.Context, cClient ccli.Client) {
+	var cg, dst string
+	var dstPtr *string
 	if len(c.Args()) < 1 {
 		ExitIfError(errors.New(strNotEnoughArgs))
 	}
 
-	cgUUID := c.Args()[0]
+	switch len(c.Args()) {
+	case 1:
+		cg = c.Args()[0]
+	default:
+		cg = c.Args()[1]
```

Comment:
Can you also add a len() check here and exit gracefully, if the number of args < expected ?